static/src/js/app/components/quick-add-button/QuickAddVocabForm.vue

```diff
@@ -224,39 +221,62 @@
       },
       setFocus() {
         this.$nextTick(() => {
-          this.$refs.select.focus();
+          // this.$refs.select.focus();
         });
       },
       // method function for parent components to reset the form when modal is closed
       resetForm() {
         this.headword = null;
         this.definition = null;
+        this.lemmaOptions = [];
+        this.lemmaId = null;
         this.showSuccesAlert = false;
-        this.showUnsuccessfullAlert = false;
+        this.showUnsuccessfulAlert = false;
         this.$store.dispatch(RESET_LATTICE_NODES_BY_HEADWORD);
       },
+      /*
+      Get the lemma options from the database by looking up the provided
+      headword as a lemma form. Runs on input to the headword field.
+       */
       async getHeadword() {
```

Comment:
This might be a good candidate to `debounce()` so that way multiple calls aren't triggered until after the user is finished typing (possible future enhancement). There's an example of this here:

https://github.com/Hedera-Lang-Learn/hedera/blob/2c5dfdba41301e0921c67deca87a4afde17f99ab/static/src/js/app/modules/LemmatizedText.vue#L86

Reply:
This seems like a good idea, but my naive attempt to implement it quickly resulted in getHeadword not running at all, so I'll add a story to the backlog for it. I don't think it'll take much to do, but I want to focus on getting stuff together for this afternoon's meeting.